src/script/interpreter.cpp

```diff
@@ -1464,14 +1464,11 @@ const HashWriter HASHER_TAPBRANCH{TaggedHash("TapBranch")};
 
 static bool HandleMissingData(MissingDataBehavior mdb)
 {
-    switch (mdb) {
```

Comment:
Imo it'd be cleaner and more readable to just leave the logic here as before, turning the wonky string assertions into comments.

Reply:
Done